src/cli.js

```diff
@@ -35,7 +35,7 @@ Options:
   --v8-cache               Emit a build using the v8 compile cache
   --license [file]         Adds a file containing licensing information to the output
   --stats-out [file]       Emit webpack stats as json to the specified output file
-  --target                  What build target to use for webpack (default: es6)
+  --target [es2015|es2020] ECMAScript target to use for output (default: es2015)
```

Comment:
```suggestion
  --target [es] ECMAScript target to use for output (default: es2015)
```

We use `[file]` as a placeholder for the others so I think this can be `[es]` as the placeholder.

In the readme we can link to the upstream docs.